### DISCUSSION TITLE
Documentação da Pinia (Gestor de Estado da Vue.js 3) (em Português).

### DISCUSSION THREAD

**Discussion by nazarepiedady** [frontendbr/forum#2344] [Mostre seu trabalho]
## Saudações, caros colegas.

Hoje eu trago uma boa noticia para todos os programadores que
trabalham sobre o ecossistema da Vue.js, a boa noticia é que fiz
a conclusão da tradução da documentação oficial da **Pinia** para
Português que pode ser acessa pela ligação abaixo:

[Documentação da Pinia (em Português)](https://pinia-docs-pt.netlify.com)

Nota de rodapé: eu também estou preparando alguns materiais para o
ecossistema da React.js, aqueles que desejarem acompanhar o meu
trabalho é só seguir o meu perfil da GitHub: [\@nazarepiedady](https://github.com/nazarepiedady/)

**Comment by felquis** [comment]
É a primeira vez que ouço falar de Pinia, mas também não uso Vue frequentemente, obrigado por compartilhar.
